os2borgerpc_client/setup.py

```diff
@@ -0,0 +1,38 @@
+from setuptools import setup
+
+setup(
+    name='os2borgerpc_client',
+    # Keep this in sync with os2borgerpc_client/jobmanager.py
+    version='0.0.5.1',
```

Comment:
I think we should bump the version to 0.1.0.0

Reply:
@agger-magenta _Should_ we bump the version number a bit more before closing this?

Reply:
We will do that when we release, i.e. when we merge to master and tag it.

Reply:
I think we're changing the numbering to ordinary semantic versioning and bumping to 1.0.0 for starters.

It's kind of mature enough already. :slightly_smiling_face: